config/ompi_check_lustre.m4

```diff
@@ -82,7 +85,7 @@ OPAL_LOG_COMMAND(
 )
     rm -f conftest.c conftest.o
     AC_MSG_RESULT([$ompi_check_lustre_struct_happy])
-
+])
```

Comment:
I think that this is indented at the wrong level -- it confusingly makes it look like you are terminating the `OMPI_CHECK_LUSTRE` macro rather than the `AS_IF`.